### DISCUSSION TITLE
difftool doesn't open

### DISCUSSION THREAD

**Discussion by mark-99** [gitextensions/gitextensions#10442] [Q&A]
Hi, just installed gitextensions, and while it mostly works both from VS 2022, and the app/shell extension, "open in difftool" does nothing. I tried installing and configuring both p4merge and meld, same (non) result. All green lights in the configure dialog.

**Comment by vbjay** [comment]
Probably would have been better to create a bug issue than a discussion. Along with following the bug template which would have asked you to provide info we need. 

https://github.com/gitextensions/gitextensions/issues/new?template=bug_report.yml

Do provide a screenshot of your config in GE or a copy of relevant config values from a git config list run.
